backend/src/index.ts

```diff
@@ -0,0 +1,109 @@
+// @TODO: configure environment variables
+// import dotenv from "dotenv";
+// dotenv.config();
+import express, { Request, Response } from "express";
+import bodyParser from "body-parser";
+import multer from "multer";
+
+const upload = multer();
+const app = express();
+const port = "3000";
+
+import passport from "passport";
+import { strategy } from "./strategy";
+
+import axios from "axios";
+import qs from "qs";
+
+const CLIENT_ID = "10000000000004";
+const CLIENT_SECRET =
```

Comment:
we can put these into a `constants.js` file for now so they can be used throughout the app